packages/twenty-front/src/pages/settings/developers/webhooks/SettingsDevelopersWebhooksNew.tsx

```diff
@@ -23,11 +23,22 @@ export const SettingsDevelopersWebhooksNew = () => {
     targetUrl: '',
     operation: '*.*',
   });
+  const [errorMessage, setErrorMessage] = useState<string | undefined>();
   const { createOneRecord: createOneWebhook } = useCreateOneRecord<Webhook>({
     objectNameSingular: CoreObjectNameSingular.Webhook,
   });
   const handleSave = async () => {
+    setErrorMessage(undefined);
+
+    try {
```

Comment:
could you use isUrl helper instead of this try catch?

Reply:
Nice 😊